app/index.tsx

```diff
@@ -1,18 +1,18 @@
 import { StatusBar } from 'expo-status-bar';
 import { StyleSheet, View, TextInput } from 'react-native';
 import Dropdown from './components/Dropdown';
+import TextInputComponent from './components/TextInputComponent';
 import NavButton from './components/NavButton';
 import PageHeader from './components/Header';
 
 export default function App() {
   return (
     <View style={styles.container}>
-      <PageHeader title='Main' pageNumber='1/4' />
-      <Dropdown label="Name" id="nameSelect" items={["Name 1", "Name 2", "Name 3", "Name 4"]} placeholder="[Select your name]" />
-      <TextInput placeholder="Team number" editable={true} />
-      <TextInput placeholder="Match number" editable={true} />
-      <Dropdown label="Driver Station Location" id="nameSelect" items={["Red 1", "Red 2", "Red 3", "Blue 1", "Blue 2", "Blue 3"]} placeholder="[Select location]" />
```

Comment:
this was an oopsy on my part but the driver station locations should be changed back to Red 1, Red 2, etc.